CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 **Added**:
 
+- **decidim-assemblies**, **decidim-participatory_processes** Table headers sortable links [\#5010](https://github.com/decidim/decidim/pull/5010)
+- **decidim-assemblies**, **decidim-conferences**, **decidim-participatory_processes** Space CTA button text changes when no components [\#5006](https://github.com/decidim/decidim/pull/5006)
```

Comment:
This line shouldn't be here

Reply:
You mean there's an empty line at  `#8`, right?

Reply:
I mean this line should be removed, as it's a duplicate form line 10

Reply:
I forgot to `git pull` and wasn't seeing it from local, hehe